src/components/Molecules/Card/Card.md

```diff
@@ -1,11 +1,11 @@
-Card
+# Card
 
 ```js
 const data = require('../../../styleguide/data/data').default;
 import Text from '../../Atoms/Text/Text';
 import Link from '../../Atoms/Link/Link';
 
-<Card image={data.image} images={data.image} backgroundColor="yellow">
+<Card imageLow={data.image} images={data.image} backgroundColor="yellow">
```

Comment:
We need add `images` ie. `images={data.images}`

Reply:
Yeah It's the props name expected by `Picture` atom

Reply:
I mean in `images={data.image}` you need to add `s` ie `image={data.images}`

Reply:
Sure